Desctiption: use packaged version of gmm instead of shipped with tarball
 Patch will have an effect, when getfem++ will be able to migrate to testing.
 Requires BD on libgmm++-dev
Author: Christophe Trophime <christophe.trophime@lncmi.cnrs.fr>
Reviewed-by: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-01-16

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -482,7 +482,13 @@
 endif(ENABLE_DINTEGRATION)
 
 if(ENABLE_GMM)
-  include_directories(contrib/gmm)
+  find_path(GMM_INC "gmm.h" PATH_SUFFIXES src include include/gmm)
+  if(GMM_INC)
+    list(APPEND EXTERNAL_INCLUDES ${GMM_INC})
+  else(GMM_INC)
+    message(STATUS "GMM " ${GMM_INC} " not Found - using contrib/gmm instead")
+    include_directories(contrib/gmm)
+  endif(GMM_INC)
   set_config_option(HAVE_GMM "Gmm")
 endif(ENABLE_GMM)
 
